Makefile

```diff
@@ -9,7 +9,7 @@ TEST_FILES=$(shell find test/*.coffee)
 .pretestem: .all
 
 .app: $(APP_FILES)
-	node_modules/.bin/browserify $(APP_FILES) -o dist/twine.js -t coffeeify
```

Comment:
Yeah, it was a little weird, because Twine got wrapped twice before. Yes, `APP_FILES` can 🔥